nemo/utils/exp_manager.py

```diff
@@ -128,6 +129,18 @@ class DLLoggerParams:
     json_file: Optional[str] = "./dllogger.json"
 
 
+@dataclass
+class ClearMLParams:
+    project: Optional[str] = None
+    task: Optional[str] = None
+    connect_pytorch: Optional[bool] = False
+    model_name: Optional[str] = None
+    tags: Optional[List[str]] = None
+    log_model: Optional[bool] = False
+    log_cfg: Optional[bool] = False
+    log_metrics: Optional[bool] = False
```

Comment:
Shall we put this into `nemo.utils.clearml_logger` too and import this in?

Reply:
Good idea

Reply:
Not nemo.utils.xyz. Please put it under Nemo.utils.loggers.xyz

Reply:
Done

Reply:
Also moved Dllogger